ecl/hql/CMakeLists.txt

```diff
@@ -166,6 +166,11 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_CLANG)
   set_source_files_properties (hqllex.cpp PROPERTIES COMPILE_FLAGS " -Wno-unused-function -Wno-error")
```

Comment:
If the code below to avoid overwriting previous flags is needed, then this line is wrong...